pkg/operator/operator_test.go

```diff
@@ -229,7 +230,7 @@ func TestOperatorSyncClusterAPIControllerHealthCheckController(t *testing.T) {
 			},
 		}
 
-		stopCh := make(<-chan struct{})
+		stopCh := make(chan struct{})
```

Comment:
Please undo

Reply:
why should not the stop channel be closed?

Reply:
undo = `s/<-//`. Channel only for reading is all that is needed.